packages/data-transformation/fhir-to-csv/src/parseFhir/parseFhir.py

```
@@ -0,0 +1,341 @@
+
+            row_count = row_count + (result_count or 0)
+    if outputFormat == 'csv':
+        csvfile.close()
+
+    elif outputFormat == 'parquet':
+        try:
+            import pyarrow as pa
+            import pyarrow.parquet as pq
+            import pandas as pd
+        except ImportError:
+            raise ImportError("Please install pyarrow and pandas to use the 'parquet' output format.")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)